tests/lax_test.py

```diff
@@ -1784,6 +1784,14 @@ def testReduceWeakType(self, op_namespace, op, arr_weak_type, init_weak_type):
       ]
       for shape, dims, strides, padding, base_dilation, window_dilation in (
         itertools.chain(
+          itertools.product(
+            [()],
+            [()],
+            [()],
+            [()],
+            [()],
+            [()]
+          ),
```

Comment:
Maybe just directly write `[((), (), (), (), (), ())]` for clarity?

Reply:
Good point! I added `VALID` and `SAME` padding values though, to test that these work on empty inputs too though, so kept the product structure.